lib/main.dart

```diff
@@ -79,7 +46,7 @@ class MyHomePage extends StatefulWidget {
 
 class _MyHomePageState extends State<MyHomePage> {
```

Comment:
consider moving the home page component to a separate file for better readability.